JavascriptFormatter.js

```diff
@@ -548,8 +543,18 @@ function formatCommand(command) {
                             eq = eq.invert();
                         line = waitFor(eq);
                     }
+                    else if (command.command.match(/^(getEval|runScript)/)) {
+                        call = new CallSelenium(def.name, xlateArgument(command.getParameterAt(0)), command.getParameterAt(0));
```

Comment:
Indeed :D Pretty sure there shouldn't evet be a second argument to those, though. Will correct it :)